client/src/components/timeline/revision.js

```diff
@@ -7,6 +7,32 @@ import Wiki from 'app/entities/wiki';
 import Date from './date';
 import Timelapse from './timelapse';
 
+// eslint-disable-next-line no-useless-escape
+const REGEX_EDIT_SUMMARY_PARTS = /(?:\/\*([^\*]+)\*\/)?(.+)?/;
+
+// Append section name to the page title if it is included in the edit summary
+const getDisplayTitle = ( title, comment ) => {
+	const matches = REGEX_EDIT_SUMMARY_PARTS.exec( comment );
+
+	if ( matches[ 1 ] ) {
```

Comment:
Since the return of [regexObj.exec](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec) is an array or `null`, this check will fail since matches is not an array.

Reply:
if you look at the regex it has 2 optional capturing groups. matches will never be null. On a no match scenario it will become ["", undefined, undefined]